dataland-frontend/tests/component/components/resources/dataTable/conversion/HighImpactClimateGetterFactory.cy.ts

```diff
@@ -34,8 +39,14 @@ describe("Unit test for the HighImpactClimateGetterFactory", () => {
     expect(value).to.have.nested.property("displayValue.label", "Applicable High Impact Climate Sectors");
     expect(value).to.have.deep.nested.property("displayValue.modalOptions.data.listOfRowContents", [
       {
-        sector: HighImpactClimateSectorsKeys[HighImpactClimateSector.NaceCodeAInGWh] ?? "",
-        energyConsumption: "12,345 GWh",
+        sector: "AGRICULTURE, FORESTRY AND FISHING",
+        energyConsumption: { value: "12345 GWh", dataSource: undefined, quality: "Estimated", comment: undefined },
+        relativeEnergyConsumption: {
+          value: "12345 GWh / €M revenue",
```

Comment:
But this is not the same value as specified in line 29 (54)

Reply:
Dood point. Fixed